manifests/charts/istiocoredns/templates/configmap.yaml

```diff
@@ -11,20 +11,19 @@ data:
     .:53 {
           errors
           health
-          {{ if eq -1 (semver  .Values.istiocoredns.coreDNSTag  | (semver "1.4.0").Compare) }}
-          # Removed support for the proxy plugin: https://coredns.io/2019/03/03/coredns-1.4.0-release/
-          grpc global 127.0.0.1:8053
+
           forward . /etc/resolv.conf {
-            except global
+            except {{- range $.Values.global.podDNSSearchNamespaces }}{{ . }} {{- end }}
           }
-          {{ else }}
-          proxy global 127.0.0.1:8053 {
-            protocol grpc insecure
-          }
-          proxy . /etc/resolv.conf
-          {{ end }}
+          
           prometheus :9153
           cache 30
           reload
         }
+
+    {{- range $.Values.global.podDNSSearchNamespaces }}
```

Comment:
I don't understand this logic. The prior logic was conditional on the tag in use. This is not conditional on the tag in use. Can you expand on what your goal with this snippet is?

Reply:
this will create multiple server blocks for each of the FQDN. I think the istiocoredns only used by multicluster cases.